ui/raidboss/data/06-ew/raid/p7s.ts

```diff
@@ -47,6 +48,43 @@ const triggerSet: TriggerSet<Data> = {
         text: 'Split Tankbusters',
       },
     },
+    {
+      // First breaks north bridge for upcoming South Knockback Spreads
+      // Second breaks remaining bridges, Separate Healer Groups
+      // Third breaks all bridges, Bait on Empty Platform
+      id: 'P7S Roots of Attis 1',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '780E', source: 'Agdistis', capture: false }),
+      condition: (data) => data.rootsCounter === undefined,
+      delaySeconds: 0.1, // Slight delay to prevent duplicate callout
```

Comment:
Sorry, can you explain what this is doing? Is it possible to do things in a more deterministic way without delays?

Reply:
The spell is the same id for each event. I'm not sure how cactbot reads trigger order, but when it was ordered as is, the first trigger would fire, set the value, second trigger fire sees the updated value, leading to multiple triggers firing "simultaneously".

It might be possible to reverse the order in the file (3 => 2 => 1) that way the 1 trigger is read last.

Reply:
I've re-ordered it, looks like the reorder of 3, 2, 1 solves the problem.

Reply:
Trigger order is deterministic, top to bottom.  The only thing that changes ordering is any asynchronicity like delaySeconds or promise.